bigbluebutton-html5/imports/api/notification/index.js

```diff
@@ -0,0 +1,2 @@
+//TODO - send the inserted notification to a notification component.
+export default new Mongo.Collection(null);
```

Comment:
`Mongo.Collection(null)` is not a database connection since it is a client side only collection
https://forums.meteor.com/t/difference-between-new-mongo-collection-null-on-client-and-new-mongo-collection-things-on-client/14401/2. The idea of using it from what I understand is to take advantage of its reactivity.